FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,14 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: self.url) { [weak self] result in
+			guard self != nil else { return }
+			switch result {
+			case let .success((data, response)):
+				completion(FeedImagesMapper.map(data, from: response))
+			default:
```

Comment:
We can improve readability by using an explicit `.failure` case, instead of `default`. Since we are dealing with a `Result` type and we don't expect it to change, we can better represent this logic by explicitly treating both the `.success` and `.failure`.
```suggestion
			case .failure:
```